doc/man1/openssl-cmp.pod.in

```diff
@@ -1023,7 +1023,7 @@ In order to enroll an initial certificate from that CA it is sufficient
 to issue the following shell commands.
 
   cd /path/to/openssl
-  export OPENSSL_CONF=openssl.cnf
+  export OPENSSL_CONF=apps/openssl.cnf
```

Comment:
The question is what is meant by /path/to/openssl. If that means the path to the openssl app or the openssl build root directory.

Reply:
Good question - I just had a look again and found that the `cd` command above it is not really needed
and it suffices to say: `export OPENSSL_CONF=path/to/openssl/apps/openssl.cnf`
which should then be clear. 
Changed this way, removing the `cd` command.